### ISSUE TITLE
Formatting in section 4.5

### ISSUE THREAD

**Issue by DanOelke** [alan412/LearnJavaForFTC#17]
# Date on version you are commenting on
May 23, 2020

# Section you are commenting on
4.5

# Comments
Missing space between of and for  "types offor loops"
I would use comma's instead of "-" in that first sentence. 
I suggest rewording "the end is done at the end of EVERY time through" to be "the update is done after EVERY time through the loop"

I also disagree on the bit about for-each.  I think that for loops are more common in the FTC code than for-each loops.  But that might be because I see the most code from teams I've mentored at least a little and as a embedded programmer I tend to use for loops more than for-each.

**Comment by alan412** [comment]
I changed the wording here.  I also use for loops more than for-each, but am trying to get my students to use for-each because they are less error prone.
